styles/main.css

```diff
@@ -0,0 +1,222 @@
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background: #fbf8ef;
+  font-family: sans-serif;
+  font-size: 24px;
+  font-weight: 900;
+}
+
+.move-up--1 {
+  transform: translateY(-85px);
+}
+.move-up--2 {
+  transform: translateY(-170px);
+}
+.move-up--3 {
+  transform: translateY(-255px);
+}
+.move-down--1 {
+  transform: translateY(85px);
+}
+.move-down--2 {
+  transform: translateY(170px);
+}
+.move-down--3 {
+  transform: translateY(255px);
+}
+.move-left--1 {
+  transform: translateX(-85px);
+}
+.move-left--2 {
+  transform: translateX(-170px);
+}
+.move-left--3 {
+  transform: translateX(-255px);
+}
+.move-right--1 {
+  transform: translateX(85px);
+}
+.move-right--2 {
+  transform: translateX(170px);
+}
+.move-right--3 {
+  transform: translateX(255px);
+}
+
+.field-cell {
+  background: #d6cdc4;
+  padding: 0;
+  width: 75px;
+  height: 75px;
+  border-radius: 5px;
+  color: #776e65;
+  box-sizing: content-box;
+  text-align: center;
+  vertical-align: center;
```

Comment:
The 'vertical-align' property does not have a value 'center'. If you want to center the text vertically with flexbox, you are already doing it with 'align-items: center;' on the '.field-cell-num' class.